charts/jaeger/templates/es-rollover-hook.yml

```diff
@@ -57,7 +57,6 @@ spec:
             - init
             - {{ include "elasticsearch.client.url" . }}
             {{ include "extra.cmdArgs" ( dict "cmdlineParams" .Values.esRollover.cmdlineParams ) | nindent 12  }}
-            {{- include "storage.cmdArgs" . | nindent 12 }}
```

Comment:
this was added to few other files also. Would you remove it from those as well then ?
Like 

- charts/jaeger/templates/es-rollover-cronjob.yaml
- charts/jaeger/templates/es-lookback-cronjob.yaml

https://github.com/jaegertracing/helm-charts/pull/414/files#diff-ec4350704d6f9fa6bbcbc2c9309e20fda060454d05bb23ad872aa97a03f46cc5
etc.

Reply:
I will test it hopefully later this day. as we are not using es-lookback atm.

Reply:
jup, same issue :)

```
jaeger-es-lookback-manual-9b6-7xsx8 Error: unknown flag: --es.num-replicas
jaeger-es-lookback-manual-9b6-7xsx8 Usage:
jaeger-es-lookback-manual-9b6-7xsx8   jaeger-es-rollover lookback http://HOSTNAME:PORT [flags]
jaeger-es-lookback-manual-9b6-7xsx8 
jaeger-es-lookback-manual-9b6-7xsx8 Flags:
jaeger-es-lookback-manual-9b6-7xsx8   -h, --help             help for lookback
jaeger-es-lookback-manual-9b6-7xsx8       --unit string      used with lookback to remove indices from read alias e.g, days, weeks, months, years (default "days")
jaeger-es-lookback-manual-9b6-7xsx8       --unit-count int   count of UNITs (default 1)
jaeger-es-lookback-manual-9b6-7xsx8 
jaeger-es-lookback-manual-9b6-7xsx8 Global Flags:
jaeger-es-lookback-manual-9b6-7xsx8       --archive                     Handle archive indices
jaeger-es-lookback-manual-9b6-7xsx8       --es.ilm-policy-name string   The name of the ILM policy to use if ILM is active (default "jaeger-ilm-policy")
jaeger-es-lookback-manual-9b6-7xsx8       --es.password string          The password required by storage
jaeger-es-lookback-manual-9b6-7xsx8       --es.tls.ca string            Path to a TLS CA (Certification Authority) file used to verify the remote server(s) (by default will use the system truststore) (default "")
jaeger-es-lookback-manual-9b6-7xsx8       --es.tls.cert string          Path to a TLS Certificate file, used to identify this process to the remote server(s)
jaeger-es-lookback-manual-9b6-7xsx8       --es.tls.enabled              Enable TLS when talking to the remote server(s) (default false)
jaeger-es-lookback-manual-9b6-7xsx8       --es.tls.key string           Path to a TLS Private Key file, used to identify this process to the remote server(s)
jaeger-es-lookback-manual-9b6-7xsx8       --es.tls.server-name string   Override the TLS server name we expect in the certificate of the remote server(s)
jaeger-es-lookback-manual-9b6-7xsx8       --es.tls.skip-host-verify     (insecure) Skip server's certificate chain and host name verification
jaeger-es-lookback-manual-9b6-7xsx8       --es.use-ilm                  Use ILM to manage jaeger indices
jaeger-es-lookback-manual-9b6-7xsx8       --es.username string          The username required by storage
jaeger-es-lookback-manual-9b6-7xsx8       --index-prefix string         Index prefix
jaeger-es-lookback-manual-9b6-7xsx8       --skip-dependencies           Disable rollover for dependencies index
jaeger-es-lookback-manual-9b6-7xsx8       --timeout int                 Number of seconds to wait for master node response (default 120)
jaeger-es-lookback-manual-9b6-7xsx8 
Stream closed EOF for jaeger/jaeger-es-lookback-manual-9b6-7xsx8 (jaeger-es-lookback)

```

```

jaeger-es-rollover-manual-qfp-5kprv Error: unknown flag: --es.num-replicas
jaeger-es-rollover-manual-qfp-5kprv Usage:
jaeger-es-rollover-manual-qfp-5kprv   jaeger-es-rollover rollover http://HOSTNAME:PORT [flags]
jaeger-es-rollover-manual-qfp-5kprv 
jaeger-es-rollover-manual-qfp-5kprv Flags:
jaeger-es-rollover-manual-qfp-5kprv       --conditions string   conditions used to rollover to a new write index (default "{\"max_age\": \"2d\"}")
jaeger-es-rollover-manual-qfp-5kprv   -h, --help                help for rollover
jaeger-es-rollover-manual-qfp-5kprv 
jaeger-es-rollover-manual-qfp-5kprv Global Flags:
jaeger-es-rollover-manual-qfp-5kprv       --archive                     Handle archive indices
jaeger-es-rollover-manual-qfp-5kprv       --es.ilm-policy-name string   The name of the ILM policy to use if ILM is active (default "jaeger-ilm-policy")
jaeger-es-rollover-manual-qfp-5kprv       --es.password string          The password required by storage
jaeger-es-rollover-manual-qfp-5kprv       --es.tls.ca string            Path to a TLS CA (Certification Authority) file used to verify the remote server(s) (by default will use the system truststore) (default "")
jaeger-es-rollover-manual-qfp-5kprv       --es.tls.cert string          Path to a TLS Certificate file, used to identify this process to the remote server(s)
jaeger-es-rollover-manual-qfp-5kprv       --es.tls.enabled              Enable TLS when talking to the remote server(s) (default false)
jaeger-es-rollover-manual-qfp-5kprv       --es.tls.key string           Path to a TLS Private Key file, used to identify this process to the remote server(s)
jaeger-es-rollover-manual-qfp-5kprv       --es.tls.server-name string   Override the TLS server name we expect in the certificate of the remote server(s)
jaeger-es-rollover-manual-qfp-5kprv       --es.tls.skip-host-verify     (insecure) Skip server's certificate chain and host name verification
jaeger-es-rollover-manual-qfp-5kprv       --es.use-ilm                  Use ILM to manage jaeger indices
jaeger-es-rollover-manual-qfp-5kprv       --es.username string          The username required by storage
jaeger-es-rollover-manual-qfp-5kprv       --index-prefix string         Index prefix
jaeger-es-rollover-manual-qfp-5kprv       --skip-dependencies           Disable rollover for dependencies index
jaeger-es-rollover-manual-qfp-5kprv       --timeout int                 Number of seconds to wait for master node response (default 120)
jaeger-es-rollover-manual-qfp-5kprv 
Stream closed EOF for jaeger/jaeger-es-rollover-manual-qfp-5kprv (jaeger-es-rollover)

```